conf.yaml

```
@@ -2278,8 +2278,7 @@ contents:
             prefix:     en/beats/functionbeat
             current:    *stackcurrent
```

`*stackcurrent` expects 8.12 to be built.
```suggestion
            current:    8.11
```